detox/src/devices/drivers/EmulatorDriver.js

```diff
@@ -117,6 +102,55 @@ class EmulatorDriver extends AndroidDriver {
     await telnet.kill();
     await this.emitter.emit('shutdownDevice', { deviceId });
   }
+
+  async _fixEmulatorConfigIniSkinNameIfNeeded(avdName) {
+    const configFile = `${os.homedir()}/.android/avd/${avdName}.avd/config.ini`;
```

Comment:
Please extract getting `configFile` to an outer method, we have a pending pull request on this, I believe. That's where you can configure from where to take config.ini.